sunpy/data/test/__init__.py

```diff
@@ -41,26 +42,45 @@ def get_test_filepath(filename, **kwargs):
     """
     return get_pkg_data_filename(filename, package="sunpy.data.test", **kwargs)
 
+
 def get_available_test_data():
     """
     Prints all available test data
 
     """
-    file_name=""
-    for fn in get_pkg_data_filenames('test', 'sunpy.data','*'):
-        if(os.path.isdir(fn)):
-            for dir_contents in os.listdir(fn):
-                print (dir_contents)
-        else:
-            for file_path in reversed(fn):
-                if(file_path == '/'):
-                    if(file_name == "__init__.py" or file_name == "bug.py"):
-                        break
+    file_name = ""
+    if(os.name == 'nt'):
+        for fn in get_pkg_data_filenames('test', 'sunpy.data', '*'):
+            if(os.path.isdir(fn)):
+                for dir_contents in os.listdir(fn):
+                    print(dir_contents)
+            else:
+                for file_path in reversed(fn):
+                    if(file_path == '\\'):
+                        if(file_name == "__init__.py"):
+                            break
+                        else:
+                            print(file_name[::-1])
+                            file_name = ""
+                            break
                     else:
-                        print (file_name[::-1])
-                        file_name=""
-                        break
-                else:
-                    file_name=file_name+file_path
+                        file_name = file_name + file_path
+    else:
+        for fn in get_pkg_data_filenames('test', 'sunpy.data', '*'):
+            if(os.path.isdir(fn)):
+                for dir_contents in os.listdir(fn):
+                    print(dir_contents)
+            else:
+                for file_path in reversed(fn):
+                    if(file_path == '/'):
+                        if(file_name == "__init__.py"):
```

Comment:
if `file_name` is going to be something that is backwards... how is going to be equal to `__init__.py`?